packages/react/src/components/Tile/Tile.tsx

```diff
@@ -238,18 +365,18 @@ export const SelectableTile = React.forwardRef(function SelectableTile(
   return (
     <div
       className={classes}
-      onClick={!disabled ? handleOnClick : null}
+      onClick={!disabled ? handleOnClick : undefined}
       role="checkbox"
       aria-checked={isSelected}
       disabled={disabled}
```

Comment:
Raises following error:

```
TS2322: Type '{ children: Element[]; defaultChecked?: boolean | undefined; defaultValue?: string | number | readonly string[] | undefined; suppressContentEditableWarning?: boolean | undefined; suppressHydrationWarning?: boolean | undefined; ... 252 more ...; title: string; }' is not assignable to type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.   Property 'disabled' does not exist on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
```

What do you propose?

Reply:
I think we should be good to remove this line since `disabled` is not a valid HTML attribute for `div` 
<img width="1209" alt="image" src="https://user-images.githubusercontent.com/40550942/233741062-9b7a5dda-2d89-47ab-b0a9-d1bbdba2c299.png">
https://www.w3schools.com/tags/att_disabled.asp

Reply:
OK then, I have also removed the other invalid attributes by the same logic (namely `value` & `name`).